src/components/Firestore/utils.ts

```diff
@@ -66,6 +66,32 @@ export function getFieldType(value: FirestoreAny): FieldType {
   return typeof value.valueOf() as FieldType;
 }
 
+export function isBoolean(value: FirestoreAny): value is boolean {
+  return !!value && typeof value.valueOf() === 'boolean';
```

Comment:
`!!value` will be `false` when `value === false`. Probably not what you want.

Reply:
Updated to using optional chaining